dev-docs/bidders/rubicon.md

```diff
@@ -42,6 +42,7 @@ For both Prebid.js and Prebid Server, the Rubicon Project adapter requires setup
 | `visitor`      | optional           | See below for details on First Party Data. In release 4.29 and earlier, this parameter allows the definition of an object defining arbitrary key-value pairs concerning the visitor for use in targeting. The values must be arrays of strings. | `{"ucat":["new"], "search":["iphone"]}`                                             | `object`         |
 | `keywords`     | optional           | See below for details on First Party Data. In release 4.29 and earlier, this can be used to influence reports for client-side display. To get video or server-side reporting, please use First Party data or the inventory/visitor parameters. | `["travel", "tourism"]`                                                             | `Array<string>`  |
 | `video`       | required for video | Video targeting parameters. See the [video section below](#rubicon-video).                                                  | `{"language": "en"}` | `object`  |
+| `bidonmultiformat` | optional | Beta parameter - please check with your account manager before setting this value | `boolean` | `true` |
```

Comment:
You may also update the

```
multiformat_supported: will-bid-on-one
```

out of curiosity: why is this not enabled by default?

Reply:
because we're testing It to make sure that multiformat actually makes money as opposed to just generating network expense for no additional revenue. the exchange cannot yet support requests for multiple formats in a single request, so by turning on multiformat, the adapters are generating multiple requests.

we'll update the docs, including this metadata, when we've run enough tests to be comfortable it's a net gain for everyone.